images/nginx/README.md

```diff
@@ -5,9 +5,6 @@
 | **Status** | stable |
 | **OCI Reference** | `cgr.dev/chainguard/nginx` |
 | **Variants/Tags** | ![](https://storage.googleapis.com/chainguard-images-build-outputs/summary/nginx.svg) |
-
-*[Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
```

Comment:
We should keep this. I think you might have an old monopod installed, can you try updating and regenerating the readme? 🙏

Reply:
Sure enough, tripped on an old version hiding in ~/go/bin/monopod - nice catch!